compact/definitions.xml

```diff
@@ -445,8 +445,11 @@ Examples:
     <comment> Forward TOF region currenlty empty for future upgrades </comment>
     <constant name="ForwardTOFRegion_zmin"        value="ForwardPIDRegion_zmin" />
     <constant name="ForwardTOFRegion_length"      value="22.0*cm" />
-    <constant name="ForwardTOFRegion_rmax"        value="CentralTrackingRegion_rmax" />
     <constant name="ForwardTOFRegion_tan"         value="CentralTrackingRegionP_tan" />
+    <constant name="ForwardTOFRegion_minR"            value="7*cm" />
+    <constant name="ForwardTOFRegion_maxR"            value="87*cm" />
```

Comment:
https://eic.jlab.org/Geometry/Detector/Detector-20220929172703.html has 8 cm to 67 cm.

Reply:
Done in new commit